bemani/utils/read.py

```diff
@@ -4103,5 +4316,24 @@ def import_music_db(self, songs: List[Dict[str, Any]]) -> None:
         danevo.import_music_db(songs)
         danevo.close()
 
+    elif series == GameConstants.GITADORA:
+        gitadora = ImportGitadora(config, args.version, args.no_combine, args.update)
+        if args.tsv is not None:
```

Comment:
I don't see a tsv file included in this diff.

Reply:
oops. i forget add this. ehh i would add tsv later. sorry~